Configurations/windows-makefile.tmpl

```diff
@@ -416,23 +416,32 @@ libclean:
 	-del /Q /F $(LIBS) libcrypto.* libssl.* ossl_static.pdb
 
 clean: libclean
-	-rmdir /Q /S $(HTMLDOCS1_BLDDIRS)
-	-rmdir /Q /S $(HTMLDOCS3_BLDDIRS)
-	-rmdir /Q /S $(HTMLDOCS5_BLDDIRS)
-	-rmdir /Q /S $(HTMLDOCS7_BLDDIRS)
 	{- join("\n\t", map { "-del /Q /F $_" } @PROGRAMS) -}
-	-del /Q /F $(MODULES)
-	-del /Q /F $(SCRIPTS)
-	-del /Q /F $(GENERATED_MANDATORY)
-	-del /Q /F $(GENERATED)
+	-"$(PERL)" -e "while (<>) { chomp; @list = split(' ', $$_); foreach $$file (@list) { unlink $$file; } }" < <<
+$(MODULES)
+$(SCRIPTS)
+$(GENERATED_MANDATORY)
+$(GENERATED)
+pod2htmd.tmp pod2htmi.tmp
+<<
```

Comment:
I obviously prefer the way it's done in #11171 :wink:

Reply:
> I obviously prefer the way it's done in #11171 wink

Feel free to merge some ideas from here with your PR, or the other way round,
you have write-access to this PR as well

Reply:
PS: I am not sure if meanwhile more files leak than before, this PR
is also about making distclean clean the build tree when it is in-tree

Reply:
I only commented on these lines, not the whole PR